content/en/apps/payments/payments-agent.md

```diff
@@ -0,0 +1,468 @@
+* Create a customer details state in the vault. [CreateCustomerDetails](#createcustomerdetails).
+* Get the balance of an account. [GetAccountBalance](#getaccountbalance).
+* Get a list of available PSPs. [GetAvailablePSPs](#getavailablepsps).
+* Get a list of available currencies available to the Payments Agent. [GetCurrenciesOnAgent](#getcurrenciesonagent).
+* Get a list of customer accounts. [GetCustomerAccounts](#getcustomeraccounts).
+* Get a payment account. [GetPaymentAccount](#getpaymentaccount).
+* Get a list of customers that you manage on the PSP. [GetPSPAccounts](#getpspaccounts).
+* Return a page of customers managed by the agent on the PSP. [GetPSPCustomers](#getpspcustomers).
+* Return the specification of the PSP. [GetPSPSpecification](#getpspspecification).
+* Return a list of transactions based on specified criteria. [GetTransactions](#gettransactions).
+* Update details of a customer by updating the details state object in the vault. [UpdateCustomerDetails](#updatecustomerdetails).
+
+## `CloseAccountDetails`
+
+Close an account state on the vault.
+
+### Parameters
+
+`accountId`. The account ID in the vault.
+
+
+### Flow Signature
+
+```
+@StartableByRPC
+@StartableByService
+@Suppress("unused")
+class CloseAccountDetails(
+    private val accountId: PaymentAccountId
+) : CloseAccountDetailsInitiator()
+```
+
+### Return Type
+
+```
+FlowLogic<Unit>
+```
+
+### Example
+
+```
+startFlow(
+    ::CloseAccountDetails,
+    accountId
+)
+```
+
+## `CloseCustomerDetails`
+
+Close a customer details state on the vault.
+
+### Parameters
+
+`customerId: PaymentCustomerId`
+
+### Flow Signature
+
+```
+@StartableByRPC
+@StartableByService
+@Suppress("unused")
+class CloseCustomerDetails(
+    private val customerId: PaymentCustomerId
+) : CloseCustomerDetailsInitiator()
+```
+
+### Return Type
+
+```
+FlowLogic<CustomerDetailsState>
+```
+
+### Example
+
+```
+startFlow(::CloseCustomerDetails, customerId)
+```
+
+## `CreateAccountDetails`
+
+Create a customer account state object in the vault.
+
+### Parameters
+
+* `accountName`. Account name.
+* `currency`. Account currency.
+* `customerId`. Existing customer who will own the account.
+* `pspAccountId`. Optional. Existing Modulr account to link to.
+
+### Flow Signature
+
+```
+@StartableByRPC
+@Suppress("unused")
+class CreateAccountDetails(
+    private val accountName: String,
+    private val currency: String,
+    private val customerId: String,
+    private val pspAccountId: String?
+) : CreateAccountDetailsInitiator()
+```
+
+### Return Type
+
+`FlowLogic<PaymentAccountDetailsState>`
+
+### Example
+
+```
+startFlow(
+    ::CreateAccountDetails,
+    accountName,
+    currency,
+    customerId,
+    pspAccountId
+)
+```
+
+## `CreateCustomerDetails`
+
+Create a payment customer details state object in the vault. If the customer already exists then a flow exception is thrown.
+
+### Parameters
+
+* `paymentCustomer`. Customer name.
+* `details`. Customer details.
```

Comment:
More detail here would help, no pun intended. This parameter should contain all the mandatory key/value pairs for the specified Customer type (i.e. business or individual ). What those mandatory fields are can be found by calling the psp spec endpoint